tests/integration/serializers/json-serializer-test.js

```diff
@@ -987,9 +987,10 @@ test('normalizeResponse ignores unmapped attributes', function(assert) {
     title: "Rails is omakase"
   };
 
-  var post = env.store.serializerFor("post").normalizeResponse(env.store, Post, jsonHash, '1', 'findRecord');
-
-  assert.equal(post.data.attributes.title, "Rails is omakase");
+  assert.expectWarning(function() {
```

Comment:
Since this tests adds an assertion for an expected warning, you need to use `testInDebug` instead of `test`, since otherwise this test will fail when the ember data tests are run in the production build.